<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1333</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>1031</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">padding-left: 15px;
padding-right: 15px;
padding-top: 5px;
padding-bottom: 5px;
margin: 5px;
font: 18pt;
</string>
       </property>
       <property name="text">
        <string>Choose File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size:15px;</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:15px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:7.8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browseButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size:15pt</string>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="imagePreview">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>671</width>
      <height>581</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(203, 203, 203, 100);
border:1px solid black;
border-radius: 10px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="applyEffect">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt;
</string>
    </property>
    <property name="text">
     <string>APPLY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt;</string>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>140</y>
      <width>581</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="4" column="1">
      <widget class="Line" name="line_4">
       <property name="styleSheet">
        <string notr="true">color:rgb(200,200,200);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="4">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item alignment="Qt::AlignHCenter">
          <widget class="QSlider" name="zoomXSlider">
           <property name="styleSheet">
            <string notr="true">width: 110px;</string>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>20</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="zoomXLabel">
             <property name="styleSheet">
              <string notr="true">padding-top: 5px;
padding-bottom: 5px;
padding-left: 5px;
padding-right:5px;
margin: 2px;
font: 20px ;</string>
             </property>
             <property name="text">
              <string>Zoom</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="zoomXValue">
             <property name="styleSheet">
              <string notr="true">padding-top: 5px;
padding-bottom: 5px;
padding-left: 5px;
padding-right:5px;
font-size:15px;
text-align: center;</string>
             </property>
             <property name="text">
              <string>10</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="3" column="2" colspan="3">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="rotationLabel">
           <property name="styleSheet">
            <string notr="true">padding-top: 5px;
padding-bottom: 5px;
padding-left: 10px;
padding-right:10px;
margin: 2px;
font: 15pt ;</string>
           </property>
           <property name="text">
            <string>Rotation</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="rotationValue">
           <property name="styleSheet">
            <string notr="true">padding-top: 5px;
padding-bottom: 5px;
padding-left: 5px;
padding-right:5px;
font-size:15px;
text-align: center;</string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QDial" name="rotationDial">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="notchTarget">
            <double>10.000000000000000</double>
           </property>
           <property name="notchesVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="Line" name="line">
         <property name="styleSheet">
          <string notr="true">color:rgb(200,200,200);</string>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="Line" name="line_2">
         <property name="styleSheet">
          <string notr="true">color:rgb(200,200,200);</string>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item alignment="Qt::AlignHCenter">
          <widget class="QSlider" name="widthSlider">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">width: 130px;</string>
           </property>
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="pageStep">
            <number>10</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="widthLabel">
             <property name="styleSheet">
              <string notr="true">padding-top: 5px;
padding-bottom: 5px;
padding-left:5px;
padding-right:5px;
margin: 2px;
font: 20px;</string>
             </property>
             <property name="text">
              <string>Width Shift</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="widthValue">
             <property name="styleSheet">
              <string notr="true">padding-top: 5px;
padding-bottom: 5px;
padding-left: 5px;
padding-right:5px;
font-size:15px;
text-align: center;</string>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="3">
        <widget class="Line" name="line_3">
         <property name="styleSheet">
          <string notr="true">color:rgb(200,200,200);</string>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="2">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item alignment="Qt::AlignHCenter">
          <widget class="QSlider" name="heightSlider">
           <property name="styleSheet">
            <string notr="true">width: 130px;</string>
           </property>
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="tracking">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item alignment="Qt::AlignLeft">
            <widget class="QLabel" name="heightLabel">
             <property name="styleSheet">
              <string notr="true">padding-top: 5px;
padding-bottom: 5px;
padding-left:5px;
padding-right:5px;
margin: 2px;
font: 18px ;</string>
             </property>
             <property name="text">
              <string>Height Shift</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="heightValue">
             <property name="styleSheet">
              <string notr="true">padding-top: 5px;
padding-bottom: 5px;
padding-left: 5px;
padding-right:5px;
font-size:15px;
text-align: center;</string>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="5">
        <widget class="Line" name="line_6">
         <property name="styleSheet">
          <string notr="true">color:rgb(200,200,200);</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">padding-top: 5px;
padding-bottom: 5px;
padding-left: 5px;
padding-right:5px;
margin: 2px;
text-align:center;
font: 15pt ;</string>
           </property>
           <property name="text">
            <string>Fill Mode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="fillMode">
           <property name="maximumSize">
            <size>
             <width>194</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size:20px;</string>
           </property>
           <item>
            <property name="text">
             <string>constant</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>nearest</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>reflect</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>wrap</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="5" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="shearLabel">
         <property name="styleSheet">
          <string notr="true">padding-top: 5px;
padding-bottom: 5px;
padding-left: 5px;
padding-right:5px;
margin: 2px;
text-align: center;
font: 15pt;</string>
         </property>
         <property name="text">
          <string>Shear</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QSlider" name="shearSlider">
           <property name="minimum">
            <number>-50</number>
           </property>
           <property name="maximum">
            <number>50</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="shearValue">
           <property name="styleSheet">
            <string notr="true">padding-top: 5px;
padding-bottom: 5px;
padding-left: 5px;
padding-right:5px;
font-size:15px;
text-align: center;</string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="6" column="1">
      <widget class="Line" name="line_10">
       <property name="styleSheet">
        <string notr="true">color:rgb(200,200,200);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Line" name="line_11">
       <property name="styleSheet">
        <string notr="true">color:rgb(200,200,200);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="Line" name="line_8">
       <property name="styleSheet">
        <string notr="true">color:rgb(200,200,200);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0" rowspan="5">
      <widget class="Line" name="line_9">
       <property name="styleSheet">
        <string notr="true">color:rgb(200,200,200);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2" rowspan="5">
      <widget class="Line" name="line_12">
       <property name="styleSheet">
        <string notr="true">color:rgb(200,200,200);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="flipCheckBoxX">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">padding-top: 5px;
padding-bottom: 5px;
padding-left: 5px;
padding-right:5px;
font-size:20px;</string>
           </property>
           <property name="text">
            <string>FlipX</string>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="flipCheckBoxY">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">padding-top: 5px;
padding-bottom: 5px;
padding-left: 5px;
padding-right:5px;
font-size:20px;</string>
           </property>
           <property name="text">
            <string>FlipY</string>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="brightnessLabel">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">padding-top: 5px;
padding-bottom: 5px;
padding-left: 5px;
padding-right:5px;
margin: 2px;
font: 15pt </string>
         </property>
         <property name="text">
          <string>Brightness</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="brightnessBox">
         <property name="styleSheet">
          <string notr="true">padding-top: 2.5px;
padding-bottom: 2.5px;
padding-left: 5px;
padding-right: 5px;</string>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>0.800000000000000</double>
         </property>
         <property name="maximum">
          <double>1.200000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>730</y>
      <width>671</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
